src/view/com/pager/FeedsTabBar.web.tsx

```diff
@@ -63,6 +67,37 @@ const FeedsTabBarDesktop = observer(
   },
 )
 
+const FeedsTabBarDesktop = observer(() => {
```

Comment:
this will now be flagged by the lint now, pls rebase :)

Reply:
Done